src/styles/blocks/calendar.scss

```
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: left;
+  padding: 10px;
+  width: 7 * ($width-day + $gap-day);
+  gap: $gap-day;
+
+  .calendar__day {
```

use nesting in all cases
```suggestion
  &__day {
```